mpu6050_scripts/poll-temp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+overwrite() { 
+	echo -e "\r\033[1A\033[0K$@"; 
+	}
+
+#Enable MPU6050 chip
+sudo i2cset -y 0 0x68 0x6B 0
```

Comment:
Is sudo required for i2c access?

Reply:
yes, sudo is required since i2c-tool package uses /dev/i2c-xx system file.
Also, if a user which runs i2c-tool added into i2c group there is not necessary to use sudo.